src/web/tools/GestureHandlerWebDelegate.ts

```diff
@@ -48,6 +44,10 @@ export class GestureHandlerWebDelegate
       this.view.style['userSelect'] = config.userSelect;
     }
 
+    this.view.style['touchAction'] = config.touchAction ?? 'none';
+    //@ts-ignore This one disables default events on Safari
+    this.view.style['WebkitTouchCallout'] = 'none';
```

Comment:
Quote from [docs](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-touch-callout):

>When a target is touched and held on iOS, Safari displays a callout information about the link. This property allows disabling that behavior.

So I think we can keep it as `none`.